tests/math_builtin_api/CMakeLists.txt

```diff
@@ -5,6 +5,13 @@ set(MATH_VARIANT base double half)
 
 set(MATH_CAT integer geometric native half)
 
+set(math_builtin_depends
+  "math_builtin.h"
```

Comment:
Should we put a header file as a dependency for source generation?

Reply:
I think you're right, it shouldn't have an impact on the generator